src/sentry/web/frontend/react_page.py

```diff
@@ -1,13 +1,19 @@
 from django.conf import settings
+from django.http import HttpResponseRedirect
 from django.middleware.csrf import get_token as get_csrf_token
 from rest_framework.request import Request
 from rest_framework.response import Response
 
+from sentry import options
 from sentry.models import Project
 from sentry.signals import first_event_pending
+from sentry.utils.http import is_using_customer_domain, query_string
 from sentry.web.frontend.base import BaseView, OrganizationView
 from sentry.web.helpers import render_to_response
 
+# url names that should only be accessible from a non-customer domain hostname.
+NON_CUSTOMER_DOMAIN_URL_NAMES = ["sentry-organization-create"]
```

Comment:
Should this also include account settings pages? One downside to forwarding these paths to the root domain will be another page load as there is a domain change.

Reply:
> One downside to forwarding these paths to the root domain will be another page load as there is a domain change.

We can avoid this by making sure we have the right links on the frontend app that avoids inducing a pageload.

Reply:
Currently all the front-end links that go to account settings are on the customer domain. I can get those changed to be on the root domain, but I think that is something we should have in place before we add redirects in.